### ISSUE TITLE
Timeline keeps empty steps regardless or removal + save

### ISSUE THREAD

**Issue by nathaneltitane** [leozide/leocad#706]
**Describe the bug**
After finishing a submodel, several empty steps remain at the end of the timeline, past the last step containing an actual part/model.

**To Reproduce**

**Expected behavior**
If last step is reached (containing an actual element in it), there should not be anything else (step) following it if it is empty

**Screenshots**
![Screenshot_20210501-124628_VNC Viewer.png](https://user-images.githubusercontent.com/5674757/116789183-a0c69400-aa7b-11eb-8d8d-beca8672697d.png)

**Version (please complete the following information):**
 - OS: Ubuntu 20.04.2 Arm64 chroot
 - LeoCAD Version : latest continuous

**Comment by leozide** [comment]
This works fine for me, if you got back a step the last one will disappear.

**Comment by nathaneltitane** [comment]
> This works fine for me, if you got back a step the last one will disappear.

Agreed, and I've done so, even after modifying the file and saving. I then closed the file and re-opened it and they were there again...

**Comment by j6t** [comment]
I can observe the problem, too. It looks like the `STEP` commands are "sticky". They are saved to the `.ldr` file even though the steps are not visible in the timeline.

To reproduce:

1. Create model with a piece in, say, Step 6. Steps 1-5 remain empty.
2. Save. Reload.
3. Move the the piece to Step 1.
4. Click button *First* (got to first step). Observe that the timeline removes steps 3-7 as expected.
5. Save. Reload.

Observe that Steps 3-6 are now visible again. Even if you fix the timeline again, the steps remain in the file.

**Comment by nathaneltitane** [comment]
still happening on latest builds
